queries/author/query.go

```diff
@@ -0,0 +1,40 @@
+package author
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"html/template"
+	"strings"
+
+	"github.com/ettle/strcase"
+)
+
+var funcMap = template.FuncMap{
+	"ToSnake": strcase.ToSnake,
+	"ToKebab": strcase.ToKebab,
+	"ToLower": strings.ToLower,
+	"ToUpper": strings.ToUpper,
+}
```

Comment:
I believe these could also come from the strings manipulation module (i think our templates are't really using them anyway so they could be deleted)